model-optimizer/extensions/front/tf/TFSliceToSlice.py

```diff
@@ -0,0 +1,76 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from extensions.ops.elementwise import Add, Equal
+from extensions.ops.select import Select
+from mo.front.common.replacement import FrontReplacementOp
+from mo.graph.graph import Graph, rename_nodes
+from mo.ops.const import Const
+from mo.ops.slice import Slice
+
+
+class TFSliceToSliceReplacer(FrontReplacementOp):
+    op = 'TFSlice'
+    enabled = True
+
+    def replace_sub_graph(self, graph: Graph, match: dict):
+        """
+        This transformation converts TFSlice to internal Slice operation.
```

Comment:
Transformation itself is not commented -- what is happening

Reply:
Moved comments to class level. Please review.